src/System.Buffers.Primitives/System/Buffers/ReadOnlyBufferExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Buffers
+{
+    public static class ReadOnlyBufferExtensions
```

Comment:
If we want to have this as an extension member, I would prefer to have it on MemoryExtensions class (an existing extensions method holder). I don't think we should be adding several extension method holders.

Reply:
We can move it to that type during corefx move.

Reply:
ok

Reply:
They are in different namespaces. MemoryExtensions is in System.